wagtailio/core/migrations/0045_update_icons_list.py

```diff
@@ -0,0 +1,1259 @@
+# Generated by Django 3.2.12 on 2022-08-19 09:11
```

Comment:
This is a massive nitpick that I feel I should apologise for asking you to rename this slightly. I prefer naming migrations to directly map to a model and what you've changed to generate or the impact. In this case, you've changed the class `SVGIcon` and as it's inheriting from `model.text_choices` I would think semantically the name of `update_svg_icon_choices` as a name is more accurate.

Reply:
Renamed - thanks Will!